CRM/Event/Form/ManageEvent.php

```diff
@@ -88,10 +88,7 @@ public function getDefaultContext() {
    * Set variables up before form is built.
    */
   public function preProcess() {
-    $config = CRM_Core_Config::singleton();
-    if (in_array('CiviEvent', $config->enableComponents)) {
-      $this->assign('CiviEvent', TRUE);
-    }
+    $this->assign('CiviEvent', CRM_Core_Component::isEnabled('CiviEvent'));
```

Comment:
Unconditional assign to align with our efforts to reduce undefined template vars.

Reply:
yay